### DISCUSSION TITLE
Toast close button position

### DISCUSSION THREAD

**Discussion by Bond-Addict** [epicmaxco/vuestic-ui#3421] [Q&A]
Does anyone know how to resolve this?
The close button for my toasts are overlapping the content.

![image001.png](https://github.com/epicmaxco/vuestic-ui/assets/117455332/653e2211-639b-4251-b2fa-5cad2377978f)

I've been trying to override the position with css but I cant seem to get it to move. 

TIA

**Comment by m0ksem** [comment]
You need to check if there are additional styles in your app that can change the behavior. For example, default vite app provides extra css that can change behavior of vuestic, like `* { positon: relative; }`

**Reply by Bond-Addict** [reply]
This was the problem. I used a sandbox vite app and then started looking at their styling. Everything works well now. Thanks for confirming my find.
